torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -463,15 +468,15 @@ std::vector<bool> IndexCompute::contiguityAnd(
 // TODO: use new mapping functions
 // This mapping might need to go through rfactor, unclear
 std::vector<bool> IndexCompute::contiguityPasC(
-    TensorDomain* producer,
-    TensorDomain* consumer) {
+    kir::TensorDomain* producer,
+    kir::TensorDomain* consumer) {
   FUSER_PERF_SCOPE("contiguityPasC");
 
   const std::vector<bool>& producer_contiguity = producer->contiguity();
   std::vector<bool> as_consumer_contiguity;
 
-  auto c_root = consumer->getRootDomain();
-  auto p_root = producer->getRootDomain();
+  auto c_root = consumer->rootDomain();
+  auto p_root = producer->rootDomain();
```

Comment:
Can we use the same name?

Reply:
which name, `getRootDomain`? For KIR in particular I'm trying make the naming more consistent (and brief) and avoid any "get" prefixes. I think we should do a similar cleanup for the Fusion IR too.

Reply:
There are a lot of `get`-prefixed functions. I'm fine with removing the prefix. I just want to make sure that's our direction and that's not just for the Kernel IR.